packages/nx-playwright/src/executors/playwright-executor/schema.json

```diff
@@ -17,6 +17,10 @@
       "type": "string",
       "description": "type of output reporter"
     },
+    "runner": {
+      "enum": ["yarn", "npx", "pnpm"],
+      "description": "runner to use to run playwright"
+    },
```

Comment:
same q  - this file contains flags for the executor, for when the tests are run.  unless I'm missing something, at this point pnpm / yarn / npm are no longer important imo